digitaltwin/e2e/tests/Microsoft.Azure.Devices.DigitalTwin.E2ETests.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.2</TargetFrameworks>
     <IsPackable>false</IsPackable>
+
+    <!-- Disabling this test project because we don't need to run it -->
+    <IsTestProject>false</IsTestProject>
```

Comment:
Since we always want to skip the PnP tests, we may as well disable them in source instead